test/features/10-functional/10-basic/40-service-details.feature

```diff
@@ -54,8 +54,21 @@ Feature: Service details page
         Then I should not see "You are ineligible if"
 
     Scenario: The address is a link to google maps
-        When I visit /service/866464
-        Then I can get to google maps by clicking "33 Elizabeth Street Richmond VIC 3121"
+       Given A service with:
+        ----------------------------------------------
+        * Location{}
```

Comment:
We need a way to do objects, arrays and values - I've used the following for now:
`Key: Value` for bare values
`Key: (nada)` for empty values
`Key:` for arrays
`Key{}` for objects